repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/config/RepairnatorConfig.java

```diff
@@ -108,6 +117,36 @@ public String getRunId() {
         return runId;
     }
 
+    public void setPipelineMode(String pipelineMode) {
+        if (pipelineMode.equals("DOCKER")) {
```

Comment:
`equals(PIPELINE_MODE.DOCKER.name())` this enforces that everything stays in sync

Reply:
Fixed.